lab-taylor/test/index-spec.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const assert = require('assert');
+var main = require(__dirname + '/../index');
+
+describe('testing module index', () => {
+  describe('testing main function', () => {
+    it('with process.argv[2] = Mya should return "hello Mya"', () => {
+      process.argv = ['fake', 'path', 'Mya'];
```

Comment:
Great Job testing the main program,, however you did not write a test for passing in an argument for greet